### PR TITLE
CNA resource status

### PR DESCRIPTION
**Pull request by jping0929**
Update CNA Gaia models to support resource status

### CODE REVIEW

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "c412242e71a24e498e4df82daf530af5de5df201",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "e500c11acdf95ded038e799636d54c9113100b49",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "282b52110672badb21c99787d5668d04162a2793",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "a6fcc3c161a4cd202ec6ddbe8fadc4ef366f3f48",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "e5b0844753faf6812c879ddcee559d4fcb2da329",
    "pipeline": "master"
  }
]
```

**Review by t00f — APPROVED** [review]

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "7a5cb15cb423ff77f1795f0745b5924d8ca8bde9",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "68dd7c48d58f6f956fdfc49146e7c41a2c1d6f8d",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "e5b0844753faf6812c879ddcee559d4fcb2da329",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "283ced18e63969acb14af0d65bc78c69dc5c7379",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "68dd7c48d58f6f956fdfc49146e7c41a2c1d6f8d",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "e5b0844753faf6812c879ddcee559d4fcb2da329",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "283ced18e63969acb14af0d65bc78c69dc5c7379",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "68dd7c48d58f6f956fdfc49146e7c41a2c1d6f8d",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "3b62ac6b80ee0866f5ea7a3b03867f6e754895e9",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "d5e8438cff9397ad957a1587dc99fea079e2095f",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "68dd7c48d58f6f956fdfc49146e7c41a2c1d6f8d",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "3b62ac6b80ee0866f5ea7a3b03867f6e754895e9",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "8fcfca9344d95cf127e555f1ebf084f93cd6f58b",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "68dd7c48d58f6f956fdfc49146e7c41a2c1d6f8d",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "3b62ac6b80ee0866f5ea7a3b03867f6e754895e9",
    "pipeline": "master"
  }
]
```

**Review by t00f — DISMISSED** [review]

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "1769b80dea9381bb907e06daba41723529e680bc",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "88739ff5f6b36195a45628bf8d8c782720a2e625",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "3b62ac6b80ee0866f5ea7a3b03867f6e754895e9",
    "pipeline": "master"
  }
]
```

File: specs/+cloudnetworkqueryfilter.spec [hunk 1]
```diff
@@ -90,6 +90,13 @@ attributes:
     - us-west-1
     omit_empty: true
 
+  - name: resourceStatus
+    description: The status of the resource.
+    type: string
+    exposed: true
+    stored: true
+    example_value: Active
+
```
Comment (rramalinga) [inline specs/+cloudnetworkqueryfilter.spec:99]: We should add omit_empty: true for resourceStatus

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "e4971fdf8fbe3be45f002c53419757074ab7e173",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "88739ff5f6b36195a45628bf8d8c782720a2e625",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "ff94719dd6f6bd2ce3ef3fe71039f15262ee1bc9",
    "pipeline": "master"
  }
]
```

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "resource-status",
    "component": "backend",
    "pr-id": "1584",
    "commit-sha": "0890ea9675a5b564c84163e5c5f538d713f47ca7",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "prismacloud",
    "pr-id": "76",
    "commit-sha": "88739ff5f6b36195a45628bf8d8c782720a2e625",
    "pipeline": "master"
  },
  {
    "project": "resource-status",
    "component": "gaia",
    "pr-id": "776",
    "commit-sha": "ff94719dd6f6bd2ce3ef3fe71039f15262ee1bc9",
    "pipeline": "master"
  }
]
```

**Review by rramalinga — APPROVED** [review]

**Pull request merged**